src/Normalizers/OpenApiNormalizer.php

```diff
@@ -15,23 +16,30 @@
 use Crescat\SaloonSdkGenerator\Helpers\NameHelper;
 use InvalidArgumentException;
 
-class OpenApiNormalizer
+final class OpenApiNormalizer
```

Comment:
What's the reason for making this final? Is there a reason this class should never be extended?

Reply:
No worries, that's fine. I'm more included to leave things extensible in OSS where people might be building custom extensions of various classes, but I'm happy to leave your changes in until someone requests that we remove them.